### DISCUSSION TITLE
Adding Structured Data using Meta Tags

### DISCUSSION THREAD

**Discussion by heksagonnet** [nuxt/nuxt#18127] [Questions] [answered]
Hi, I am trying to add Schema.org's JSON-LD [structured data](https://developers.google.com/search/docs/advanced/structured-data/intro-structured-data) into the head section. I tried to implement the following: 
```html
    <Html>
      <Head>
        <Script type="application/ld+json">
          {"@context":"https://schema.org","@type":"Organization","name":"Heksagon"}
        </Script>
      </Head>
    </Html>
```

Which rendered out the following at the head section: 
```html
<script type="application/ld+json"></script>
```

Is this a bug or an implementation error on my part? Appreciate any help to provide some clarity in my use case.

Regards.

**Comment by viii47** [comment, marked as answer]
See https://github.com/nuxt/framework/discussions/2220#discussioncomment-1722493

**Reply by heksagonnet** [reply]
Thanks @viii47
